style.css

```diff
@@ -0,0 +1,593 @@
+  font-weight: 400;
+}
+
+.items ul li:nth-child(3) {
+  color: #7a869a;
+  margin-left: 10px;
+  margin-right: 10px;
+  font-weight: 400;
+}
+
+.btn {
+  border: none;
+  padding: 4px 12px;
+  color: #6070ff;
+  text-decoration: none;
+  background-color: #ebebff;
+  border-radius: 10px;
+  margin-right: -20px;
+  font-weight: 500;
+}
+
+.btn-div {
+  margin: 15px 0;
+  list-style: none;
+  font-size: 12px;
+  display: flex;
+  flex-direction: row;
+}
+
+.last-btn {
+  border: 1px solid #6070ff;
+  padding: 10px 18px;
+  border-radius: 10px;
+  margin-top: 8px;
+  background-color: white;
+  color: #6070ff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  transition: ease-in 0.4s all;
+}
+
+.last-btn:hover {
+  background: #4053fc;
+  color: #fff;
+  cursor: pointer;
+}
+
+.last-btn:active {
+  background: rgba(34, 48, 210, 1);
+  color: #fff;
+  cursor: pointer;
+}
+
+.section-3 {
+  background-color: #f7f7f9;
+}
+
+.container-3 {
+  background-color: white;
+  border-top-right-radius: 100px;
+  padding-left: 30px;
+  padding-top: 100px;
+  padding-right: 20px;
+  margin-bottom: 195px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.about {
+  font-size: 2.4em;
+  padding-bottom: 10px;
+}
+
+.about-div p {
+  padding-bottom: 15px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-size: 16px;
+}
+
+.m-b {
+  margin-top: 20px;
+}
+
+.languages {
+  padding-top: 50px;
+}
+
+.languages-div {
+  display: flex;
+  align-items: center;
+  padding-left: 20px;
+  list-style: none;
+}
+
+.framework {
+  border-bottom: 1.5px solid #dfe1e6;
+  padding: 25px 0;
+  display: flex;
+  justify-content: space-between;
+  padding-right: 60px;
+}
+
+.skills {
+  background-color: #f7f7f9;
+  margin-top: 20px;
+  padding-bottom: 15px;
+  padding-top: 15px;
+  border-radius: 10px;
+  margin-right: 20px;
+  display: flex;
+}
+
+.m-l {
+  margin-left: 16px;
+}
+
+.language {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 60px;
+}
+
+.section-2-desktop {
+  display: none;
+}
+
+.section-4 {
+  background-color: #6070ff;
+  background-image: url("Media/contact\ form\ shapes-mobile@2x.png");
+  background-position: right 0;
+  background-repeat: no-repeat;
+  width: 100%;
+  border-top-left-radius: 100px;
+}
+
+.section-4-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.full-name {
+  margin-bottom: 2rem;
+}
+
+.contact {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+  margin-top: 80px;
+  margin-bottom: 10px;
+}
+
+.section-4-container p {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #fff;
+  margin-bottom: 20px;
+  margin-top: 10px;
+  padding: 0 20px;
+}
```

Comment:
Please consider using the same `color and width properties `used in the Figma Design to make it closer to the design. 

```
.section-4-container p {
  color: #EBEBFF;
  width: CHECK DESIGN
}
```